openfl-workspace/torch_cnn_mnist/plan/plan.yaml

```diff
@@ -28,6 +28,8 @@ data_loader :
 task_runner :
   defaults : plan/defaults/task_runner.yaml
   template : code.pt_cnn.PyTorchCNN
+  settings :
+    loss :
```

Comment:
@mansishr I think it would be worth commenting out `loss`, and also provide an example of acceptable values. i.e.:

```    
       settings :
           # built-in loss functions from torch.nn
           # loss : L1Loss
```